packages/shared/src/components/modals/post/CreateSharedPostModal.tsx

```diff
@@ -72,41 +71,44 @@ export function CreateSharedPostModal({
     checkUrl(value);
   };
 
+  const footer = (
+    <>
+      <Button
+        icon={<AtIcon />}
+        className="btn-tertiary"
+        size={ButtonSize.Small}
+        onClick={markdownRef?.current?.onMentionCommand}
+      />
+      <Divider vertical />
+      <SourceButton source={squad} size="small" />
+      <span className="-ml-1 flex-1">
+        <strong>{squad.name}</strong>
+        <span className="ml-1 text-theme-label-tertiary">@{squad.handle}</span>
+      </span>
+    </>
+  );
+
+  const submitProps = {
+    color: ButtonColor.Cabbage,
+    variant: ButtonVariant.Primary,
```

Comment:
Would this not get overwritten by the one set on the renderer?

Reply:
It was just the props was not accepting just the color.

Reply:
Oh right, so it needs this here, even though it doesn't really do anything?

Reply:
Yes, the button typescript is shouting when you add color without the variant due to the type below.

![Screenshot 2024-03-20 at 6 14 59 PM](https://github.com/dailydotdev/apps/assets/13744167/2102bdc9-ee03-4f63-9d13-4b0d535b224b)